### PR TITLE
Fix #69: Remove test packages from supression on inspect tools.

### LINKED ISSUE #69: Activate inspect tools for tests package
Actually they don't run on test packages

### PR DESCRIPTION
**Pull request by raulccabreu**
- Remove test packages from supression
- Add <suppress checks="VisibilityModifier" files="Test" /> to silence @rules public members on test classes.
- Decrease inspect line length permitted size from 120 to 100.
- Fix checkstyle problems found in test packages.
- Rename idlings package to lockers.

### CODE REVIEW

File: app/src/androidTest/java/br/com/catbag/gifreduxsample/ui/giflist/GifListActivityTest.java [hunk 1]
```diff
@@ -24,8 +24,12 @@
 import br.com.catbag.gifreduxsample.MyApp;
 import br.com.catbag.gifreduxsample.R;
 import br.com.catbag.gifreduxsample.asyncs.data.DataManager;
+<<<<<<< Updated upstream
 import br.com.catbag.gifreduxsample.asyncs.net.downloader.FileDownloader;
 import br.com.catbag.gifreduxsample.idlings.AnvilTestLocker;
+=======
+import br.com.catbag.gifreduxsample.lockers.AnvilTestLocker;
+>>>>>>> Stashed changes
```
Comment (niltonvasques) [inline app/src/androidTest/java/br/com/catbag/gifreduxsample/ui/giflist/GifListActivityTest.java]: Merge failed!

**Review by niltonvasques — CHANGES_REQUESTED** [review]

File: app/src/androidTest/java/br/com/catbag/gifreduxsample/ui/giflist/GifListActivityTest.java (hunk 1 above)
Comment (raulccabreu) [reply app/src/androidTest/java/br/com/catbag/gifreduxsample/ui/giflist/GifListActivityTest.java]: NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO!

File: app/src/main/java/br/com/catbag/gifreduxsample/models/Gif.java [hunk 2]
```diff
@@ -14,18 +14,15 @@
...
 
     @Value.Default
     public String getPath() {
         return "";
     }
 
-    @Value
     public abstract String getUrl();
-
-    @Value
```
Comment (niltonvasques) [inline app/src/main/java/br/com/catbag/gifreduxsample/models/Gif.java:28]: I guess that uses @Value.Immutable let more explicit the immutability from those methods.

**Review by niltonvasques — CHANGES_REQUESTED** [review]

File: app/src/main/java/br/com/catbag/gifreduxsample/models/Gif.java (hunk 2 above)
Comment (raulccabreu) [reply app/src/main/java/br/com/catbag/gifreduxsample/models/Gif.java:28]: They aren't permited to be used on methods...i tried here. :(
I can put a comment like `//abstract methods are immutables by default`

Comment (niltonvasques) [reply app/src/main/java/br/com/catbag/gifreduxsample/models/Gif.java:28]: I got it! The comments will help.

Comment (raulccabreu) [reply app/src/main/java/br/com/catbag/gifreduxsample/models/Gif.java:28]: I put on first method only to not pollute the class with tree equals comments.

**Review by niltonvasques — APPROVED** [review]

**Comment by niltonvasques** [comment]
:+1:

**Pull request merged**